app/site/app/views/layouts/site/application.html.slim

```diff
@@ -3,7 +3,7 @@ html lang="en"
 head
   meta charset="utf-8"
 
-  = javascript_include_tag "local_cdn/react-0.12.2", "local_cdn/JSXTransformer-0.12.2"
+  = javascript_include_tag "local_cdn/react-0.12.2", "local_cdn/JSXTransformer-0.12.2", "local_cdn/fluxxor"
```

Comment:
Why don't we add these to the asset pipeline? (config/assets.rb) or is it not possible inside Engine?